maintainers/README.md

```diff
@@ -36,17 +36,60 @@ Issues on the board progress through the following states:
 
 - No Status
 
-  Team members can add pull requests or issues to discuss or review together.
-
   During the discussion meeting, the team triages new items.
+  To be considered, issues and pull requests must have a high-level description to provide the whole team with the necessary context at a glance.
+
+  On every meeting, at least one item from each of the following categories is inspected:
+
+  - [critical](https://github.com/NixOS/nix/labels/critical)
+  - [security](https://github.com/NixOS/nix/labels/security)
+  - [regression](https://github.com/NixOS/nix/labels/regression)
+  - [bug](https://github.com/NixOS/nix/issues?q=is%3Aopen+label%3Abug+sort%3Areactions-%2B1-desc)
+
+  - [oldest pull requests](https://github.com/NixOS/nix/pulls?q=is%3Apr+is%3Aopen+sort%3Acreated-asc)
+  - [most popular pull requests](https://github.com/NixOS/nix/pulls?q=is%3Apr+is%3Aopen+sort%3Areactions-%2B1-desc)
+  - [oldest issues](https://github.com/NixOS/nix/issues?q=is%3Aissue+is%3Aopen+sort%3Acreated-asc)
+  - [most popular issues](https://github.com/NixOS/nix/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc)
+
+  Team members can also add pull requests or issues they would like the whole team to consider, picking from the following topics:
+
+  <!-- reviewers: please weed out what we should ignore for now -->
+
+  - [installer](https://github.com/NixOS/nix/labels/installer)
+  - [nix-shell](https://github.com/NixOS/nix/labels/nix-shell)
+  - [documentation](https://github.com/NixOS/nix/labels/documentation)
+  - [UX](https://github.com/NixOS/nix/labels/UX)
+  - [error-messages](https://github.com/NixOS/nix/labels/error-messages)
+  - [new-cli](https://github.com/NixOS/nix/labels/new-cli)
+  - [flakes](https://github.com/NixOS/nix/labels/flakes)
+  - [developer-experience](https://github.com/NixOS/nix/labels/developer-experience)
+  - [macos](https://github.com/NixOS/nix/labels/macos)
+  - [windows](https://github.com/NixOS/nix/labels/windows) (new)
+  - [performance](https://github.com/NixOS/nix/labels/performance) (new)
+  - [ca-derivations](https://github.com/NixOS/nix/labels/ca-derivations)
```

Comment:
Do we really need this list here? Team members can bring up whatever PRs/issues they want. They don't have to have any particular label.

Reply:
You're right. We don't require the list, it was meant as an overview. In practice we will probably mostly pick new things anyway, and label those in the process.

We could use the later though, if we decided to focus on a selection of topics, and add those to the categories to triage every week. Will remove it.